internal/controller/cache/kind.go

```diff
@@ -14,16 +14,16 @@ import (
 const IDIndex = "id"
 
 type kindIndices struct {
-	mu sync.RWMutex
```

Comment:
Is swapping these two types in the struct definition what causes the size to decrease?